redis/commands/core.py

```diff
@@ -3255,6 +3256,25 @@ def bzpopmin(self, keys, timeout=0):
         keys.append(timeout)
         return self.execute_command("BZPOPMIN", *keys)
 
+    def zmpop(
+        self,
+        num_keys: int,
+        keys: List[str],
+        min_max: str,
```

Comment:
Instead of min_max we should do min=False, max=False and then check and raise a DataError.